packages/valory/services/hello_world/service.yaml

```diff
@@ -9,6 +9,7 @@ fingerprint:
 fingerprint_ignore_patterns: []
 agent: valory/hello_world:0.1.0:bafybeihuylwbdvru6evry6dv7r7mfuaasjclo3f5vnyes6cobkedybsqwy
 number_of_agents: 4
+deployment: {}
```

Comment:
@0xArdi @Adamantios @8ball030 You can define your deployment configurations here, for now you can define agent ports as 

```yaml
deployment:
  agent:
    ports:
      0:
        8080: 8080
      1:
        8080: 8081
      2:
        8080: 8082
      3:
        8080: 80837
```

The deployment config for docker an k8s will still look like https://github.com/valory-xyz/open-autonomy/pull/1882#discussion_r1135025858